standards/peer-review.md

```diff
@@ -1,41 +1,38 @@
-  - Reviewers SHOULD NOT use the "request changes" option on draft PR reviews
-  - When a draft PR is ready for full review the requesting engineer MUST update its status and request (or re-request) review from relevant team members
+- **PR reviews are a core responsibility**
+  This isn't an "extra" task; it's a fundamental part of every engineer's job, regardless of level. Actively participating in reviews is crucial for code quality, knowledge sharing, and team success.
 
-# Peer Review
+- **Be timely & responsive**
+  Prioritize reviews to keep the development flow moving. Aim for initial feedback within a **24-48 hour** timeframe. If you have thoughts but can't give a full review, consider setting up a 30-minute walk-through with the author. After you have pushed updates based on feedback, **re-request a review** from the original reviewers to notify them that the PR is ready for another look. This helps everyone track the status and keeps the process moving.
 
-- All peer review MUST comply with our [Engineering Values](../culture/values.md)
+- **Be constructive & kind**
+  Focus on the code, not the person. Frame feedback as suggestions ("What do you think about trying...?") and questions ("Is there a reason we chose this approach?"). Avoid demands. Use "we" or "us" language where appropriate to foster a sense of shared ownership.
 
-- All work SHOULD go through the process of peer review. It is a form of collaboration and it strengthens quality.
+- **Understand the goal**
+  The PR author should provide a descriptive title and sufficient context in the PR description. As a reviewer, review the code against its stated purpose and acceptance criteria. Does the code achieve what it set out to do?
 
-- It SHOULD involve at least one (1) more experienced developer evaluating whether best practices are followed.
+- **Check for clarity & maintainability**
+  Is the code easy to understand? Are variable and function names clear? Will future developers be able to work with this? Look for opportunities to simplify, clarify, or refactor. Good code is not just clever; it's maintainable.
 
-- It MAY consist of at least one (1) session of one-on-one peer review.
+- **Verify functionality & tests**
+  Does the code work as expected? Are there adequate tests covering new functionality and edge cases? When necessary, pull down the branch and run the code locally to verify its behavior.
 
-  - Example: One team member would make an appointment with another team member to chat about coding progress, followed by walk-through of code-in-review.
+- **Encourage small, focused PRs**
+  Smaller PRs are easier and faster to review, lead to more thorough feedback, and reduce risk. Reviewers can and should request that large changes be broken down into smaller, logical PRs as a requirement for approval.
 
-- It MAY involve a team member who is less experienced with the tech stack.
+- **Use Draft PRs for early feedback**
```

Comment:
Some teams use labels such as "In Progress" to indicate that a PR is a draft and is still being worked on. There's also "Feedback Wanted". Can we add this as an approach and open it so it's along the lines of "use your team's agreed upon conventions..."?